recipes/jq-py/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  name: jq.py
```

Comment:
Usually, when there is a name clash between a non-python package and its python counterpart, we go for `python-name` instead. However, I see that this is named `jq.py` in the GH repo. So I guess it is OK.